e2e/utils/command-utils.ts

```diff
@@ -70,7 +70,7 @@ export function runCommand(
 ): string {
   const { failOnError, ...childProcessOptions } = options ?? {};
   try {
-    const r = execSync(`${command}${isVerbose() ? ' --verbose' : ''}`, {
```

Comment:
unsure if this is supposed to be removed or just updated

Reply:
This was added a few days ago and breaks tests that run commands which explicitly do not allow a `verbose` flag - e.g. `git ...`